gdal/doc/source/api/index.rst

```diff
@@ -112,7 +112,8 @@ API
        Go <https://github.com/lukeroth/gdal>
        Julia <https://github.com/JuliaGeo/GDAL.jl>
        Lua <https://trac.osgeo.org/gdal/wiki/GdalOgrInLua>
-       Node.js <https://github.com/naturalatlas/node-gdal>
+       Original Node.js bindings <https://github.com/naturalatlas/node-gdal>
+       Node.js fork with full Promise-based async and TypeScript support <https://www.npmjs.com/package/gdal-async>
```

Comment:
@mmomtchev why not adding https://github.com/contra/node-gdal-next? it also seems maintained

maybe we could add https://github.com/ddohler/gdal-js/ and https://github.com/azavea/loam just for sake of listing all available options 🤷‍♂️

Reply:
Do you want me to add them?

I used to be the main contributor to node-gdal-next (of which node-gdal-async is a fork)
The other two are browser projects - which is very different.

Reply:
Is there a subject that needs to be raised with OSGeo? OSGeo is about excellence in software and any other subjects do not belong here.